storage/Storage.cc

```diff
@@ -5878,6 +5878,8 @@ namespace storage
 	ConstDiskIterator i = findDisk( disk );
 	if( i != dEnd() )
 	{
+            info.dasd_format = storage::DASDF_NONE;
```

Comment:
I suggest to move those initial values to Disk::getInfo() so it is always properly initialized.
The overwritten Dasd::getInfo() will return the real values for DASDs.